released/packages/coq-mathcomp-multinomials/coq-mathcomp-multinomials.2.2.0/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+maintainer: "pierre-yves@strub.nu"
```

Comment:
I should probably replace this line with my e-mail address since we (MathComp dev) agreed that I will maintain multinomials, but I will figure it out later.